### ISSUE TITLE
Update API Provider quickstart to use OpenAPI 3 specification

### ISSUE THREAD

**Issue by MelissaFlinn** [syndesisio/syndesis-quickstarts#47]
Apicurio and syndesis now support OpenAPI 3. API Provider example should probably also be updated.

**Comment by zregvart** [comment]
From what I see we need a OpenAPI 3 version of [task-api.json](https://github.com/syndesisio/syndesis-quickstarts/blob/master/api-provider/task-api.json) and re-export the [TaskAPI-export.zip](https://github.com/syndesisio/syndesis-quickstarts/blob/master/api-provider/TaskAPI-export.zip).

**Comment by MelissaFlinn** [comment]
Thank you!

**Comment by TovaCohen** [comment]
@zregvart 
For 7.5, task-api.json is available at
https://raw.githubusercontent.com/syndesisio/syndesis-quickstarts/1.8/api-provider/task-api.json
and
TaskAPI-export.zip is available at 
https://github.com/syndesisio/syndesis-quickstarts/tree/1.8/api-provider

And both those URLs are in the user doc.

For 7.6, do I just need to update those URLs to replace "1.8" with "1.9"?

**Comment by zregvart** [comment]
@TovaCohen yes, I've just created the [1.9 tag](https://github.com/syndesisio/syndesis-quickstarts/releases/tag/1.9).

**Comment by TovaCohen** [comment]
Thanks Zoran. I ensured that the URLs in the Fuse Online doc have 1.9 now. 
But, when I imported the 1.9 TaskAPI-export.zip file, for each operation flow, the settings of the error conditions are not correct: 

GET  error conditions are set to "200 All is good"
POST  error conditions are set to "200 All is good"
GET/id  errorKeys.SqlDuplicateKeyError set to "200 All is good". Settings for other errors seem correct
PUT/id  errorKeys.SqlDuplicateKeyError set to "200 All is good". Settings for other errors seem
DELETE/id  errorKeys.SqlDuplicateKeyError set to "204 Task deleted". Settings for other errors seem

This is a bit different from a similar problem with imported return codes in the previous release. See https://issues.redhat.com/browse/ENTESB-11882 which Kurt fixed for 7.5. Let me know if you want me to create an ENTESB Jira to get this corrected.
